triggers/runner/BUILD.bazel

```diff
@@ -28,16 +29,22 @@ da_scala_library(
         "//daml-lf/language",
         "//daml-lf/transaction",
         "//daml-script/converter",
-        "//language-support/scala/bindings",
-        "//language-support/scala/bindings-pekko",
+        "//ledger-api/grpc-definitions:ledger_api_proto_scala",
```

Comment:
that won't work when rebased on main, same througout this PR, this needs to be `//canton:ledger_api_proto_scala`

Reply:
:+1: